### PR TITLE
Array Declaration: handle keyless, parenthesis less closures as array values

### PR DESCRIPTION
**Pull request by mikesherov**

### CODE REVIEW

File: Behance/Sniffs/Arrays/ArrayDeclarationSniff.php [hunk 1]
```diff
@@ -279,7 +278,15 @@ protected function _handleSingleLineArray($phpcsFile, $arrayStart, $arrayEnd) {
 
   }
 
-  private function _isNestedComma($tokens, $stackPtr, $nextToken) {
+  private function _findStartOfStatementOrScope(PHP_CodeSniffer_File $phpcsFile, int $stackPtr) {
```
Comment (moskalen) [inline Behance/Sniffs/Arrays/ArrayDeclarationSniff.php]: not blocking: you don't use return type in php sniff, do you?

Comment (mikesherov) [reply Behance/Sniffs/Arrays/ArrayDeclarationSniff.php]: I should, thanks.

Comment (mikesherov) [reply Behance/Sniffs/Arrays/ArrayDeclarationSniff.php]: really old bad habit... this is an old codebase, so I've been following the old style.

Comment (mikesherov) [reply Behance/Sniffs/Arrays/ArrayDeclarationSniff.php]: fixed. thanks for the nudge

**Pull request merged**